llvm/lib/Support/Signals.cpp

```diff
@@ -253,6 +253,117 @@ static bool printSymbolizedStackTrace(StringRef Argv0, void **StackTrace,
   return true;
 }
 
+#ifdef LLVM_ENABLE_DEBUGLOC_COVERAGE_TRACKING
+void sys::symbolizeAddresses(AddressSet &Addresses,
+                             SymbolizedAddressMap &SymbolizedAddresses) {
+  assert(!DisableSymbolicationFlag && !getenv(DisableSymbolizationEnv) &&
+         "Debugify origin stacktraces require symbolization to be enabled.");
```

Comment:
This will want all kinds of caveats to discourage people from using it for any other purpose